missingNumbers.js

```diff
@@ -0,0 +1,25 @@
+function findMissingNumbers(inputArray) {
+
+    var missingNumbers = [];
+    if (typeof inputArray === 'string') {
+        return 'invalid input'
+    }
+    if (inputArray.length <= 1) {
+        return 'array length too small'
+    }
+    inputArray = inputArray.sort()
+    var i = 1
+    var missingValue = inputArray[0] + 1;
+    const count = ((inputArray[inputArray.length - 1]) - missingValue) - (inputArray.length - 2);
```

Comment:
A few questions on this?
- What count doe this variable keep track of?
- Why do you do this `inputArray.length - 2`, How did you come up with 2